src/pages/home/home.scss

```diff
@@ -0,0 +1,77 @@
+@import "~theme/base/variables";
+
+.homeContainer {
+  height: 100%;
+}
+
+.navigationBar {
+  padding: 1.25em 0em;
+}
+
+.leftNavLinks {
+  margin-left: 4em;
+}
+
+.navLinks {
+  display: inline-block;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #179A6F;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 1rem;
```

Comment:
There's a PR that adds font sizes, I'm just waiting for the author to resolve the conflicts to merge. Ideally, we should use those variables here.

Reply:
I left one comment on that PR then I'll merge!

Reply:
Yes, ideally. However, since this was not int the codebase when I started, I left it out as I saw an incoming PR with those additions.

Reply:
PR #89 has been merged now. You can resolve the conflicts and update it as per your need. I will review all the CSS after you are finally done and let you know if there is anything missing and can be replaced with variables. Thanks!

Reply:
We can merge that in later also! Thanks @arthurbuhl  We don't have to get it right 100% right now, let's get the ball rolling